extensions/database-documentation/.eslintrc.json

```diff
@@ -0,0 +1,9 @@
+{
+    "parserOptions": {
+		"project": "./extensions/database-documentation/tsconfig.json"
+	},
+	"rules": {
+		// Disabled until the issues can be fixed
```

Comment:
Please don't disable this - all new extensions should be fixing issues right away instead